lib/shopify_app/configuration.rb

```diff
@@ -26,6 +28,14 @@ def has_webhooks?
     def has_scripttags?
       scripttags.present?
     end
+
+    def has_fulfillment_services?
+      fulfillment_services.present?
+    end
+
+    def has_base_url?
+      base_url.present?
+    end
```

Comment:
hmm thinking about this again perhaps we should have the base url option. Can you split that into a new PR though and have this one be explicit? 

If we go this route we should make all the services use this variable and add it to the generator and the readme